### PR TITLE
Add how-to for enabling HTTPS

### LINKED ISSUE #4094: Gitea setup using HTTPS
To force Gitea to use HTTPS I had to use Google as such an item I did not see in the documentation, it can be there and you will tell me where it is, and so I think it needs to be created

### PR DESCRIPTION
**Pull request by jonasfranz**
Document who to enable HTTPS for Gitea.

Fixes #4094.

### CODE REVIEW

**Review by techknowlogick — APPROVED** [review]

**Comment by techknowlogick** [comment]
@JonasFranzDEV This isn't a blocking request, but could you add a note about `REDIRECT_OTHER_PORT` and `PORT_TO_REDIRECT`, and how users can set up a redirect from HTTP to HTTPS.

**Comment by lunny** [comment]
LGTM. @techknowlogick that could be another PR.

File: docs/content/doc/usage/https-support.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+```
+To learn more about the config values, please checkout the [Config Cheat Sheet](../config-cheat-sheet#server).
+
+## Using reverse proxy
+
+Setup up your reverse proxy like shown in the [reverse proxy guide](../reverse-proxies).
+
+After that, enable HTTPS by following one of these guides:
+
+* [nginx](https//nginx.org/en/docs/http/configuring_https_servers.html)
```
Comment (tylerlm) [inline docs/content/doc/usage/https-support.md:41]: There is a missing colon on line 41 in the nginx URL, `https//` should be `https://`

Comment (techknowlogick) [reply docs/content/doc/usage/https-support.md:41]: @tylerlm thanks for pointing this out. I've created a PR to fix this.

**Pull request merged**